dist/tools/travis-scripts/build_and_test.sh

```diff
@@ -64,7 +64,7 @@ then
 
     source ./dist/tools/pr_check/check_labels.sh
 
-    if check_gh_label "Ready for CI build"; then
+    if check_gh_label "Ready for CI build" || test "$SKIP_CI_READY_CHECK" = "yes"; then
```

Comment:
reverse the order so that we don't need to do a HTTP request if we are ignoring the result anyway.